next-app/src/lib/debounce.ts

```diff
@@ -0,0 +1,7 @@
+let id: ReturnType<typeof setTimeout>
+
+export function debounce(fn: () => void, delay_in_ms: number) {
```

Comment:
#lodash 😄

Reply:
I looked into the just-debounce I believe but still needed to "reset" with my own code so it didn't make sense but you're right, if we had pulled in lodash, it might've already been there to serve the purpose.  FWIW, if I'd been able to get it to bundle properly, it would be in already!